### PR TITLE
Simplify revenue tiles + default Total Fees chart to all-time

### LINKED ISSUE #826: Earned-vs-Collected chart: tonexty fill inverts if collected crosses above earned
## Problem

The Earned-vs-Collected cumulative chart (`ui-dashboard/src/app/revenue/_components/borrowing-earned-collected-chart.tsx`, PR #823) shades the band between the collected and earned traces with Plotly `fill: "tonexty"` to represent the outstanding receivable. If cumulative collected ever exceeds cumulative protocol-share earned at some timestamps — possible transiently from FX-pricing drift between the legs (collected is priced at today's oracle rate, earned accrues the same way, but rounding/split-sentinel windows can cross) — the traces cross and `tonexty` fills on both sides of the crossing, which reads as a negative/inverted receivable band. Low severity (no real data shows this today; the summary tile already clamps `receivableUSD` at 0), deferred from PR #823 review as a visual edge case.

## Solution ideas

- Clamp the earned trace's fill companion: render the band against `max(collected, earned)`/`min(...)` helper traces so the shaded area is always the non-negative receivable, or
- Split into three traces: collected (solid), earned (line), plus an explicit receivable band trace computed as `max(0, earned − collected)` stacked on top of collected, or
- Simplest: accept crossing visually but add a hover/legend note; revisit only if real data ever crosses.

### PR DESCRIPTION
**Pull request by chapati23**
## The Problem

- The Total Fees chart defaults to a 30-day window, so its headline ($1.8K) sat directly under the all-time Swap Fees tile ($5.1K) and read as "total is lower than one of its parts".
- The Earned-vs-Collected chart renders two visually identical lines on real data — Celo's rebalancer touches troves constantly, so collected tracks earned within ~0.1% — and the CDP tile's collected/accruing figures + progress bar added noise without daily signal.

## The Solution

Default the Total Fees chart to all-time so it agrees with the tiles above it, drop the Earned-vs-Collected chart, and slim the revenue tiles down to one context row and one-line subtitles. The collected/receivable data stays fully indexed — only the rendering goes.

## Details

- `fee-over-time-chart.tsx`: default range `"30d"` → `"all"`; the 1W/1M buttons still work.
- `borrowing-earned-collected-chart.tsx` deleted + its render/tests removed. `LiquityInstance.borrowingFeeCollectedCum` and `LiquityBorrowingRevenueDailySnapshot.collected` remain indexed and queryable.
- CDP Borrowing Fees tile: protocol-share headline, one row `Gross $X · of which $Y goes to stability pool`, subtitle `Protocol share of borrowing fees`. Approximate/loading/error states unchanged.
- Swap Fees tile subtitle → `Protocol share of swap fees`.
- Closes #826 (tonexty fill edge case — obsolete with the chart removed).

## Validation

- `pnpm typecheck`, `pnpm lint`, 3,032 vitest tests, react-doctor full score — all green.
- Browser-verified on a local dev server against live production data: tiles render `$1.1K earned | Gross $4.5K · of which $3.4K goes to stability pool | Protocol share of borrowing fees`; Total Fees opens on All at $6.2K (= $5.1K swap + $1.1K borrowing share, matching the feeRecipient Safe within 0.3%); zero console errors.

## Deferrals

- None

🤖 Generated with [Claude Code](https://claude.com/claude-code)

---

> [!NOTE]
> **Low Risk**
> Dashboard-only copy and layout changes; fee data hooks and indexing are untouched, with tests updated for the new UI.
> 
> **Overview**
> Simplifies the **Protocol Revenue** dashboard so all-time summary tiles and the **Total Fees** chart tell a consistent story, and removes low-signal earned-vs-collected UI.
> 
> **Total Fees** chart now opens on **all-time** (`"all"`) instead of 30 days, so its headline aligns with the all-time swap and CDP tiles above it. Range buttons (1W / 1M / etc.) are unchanged.
> 
> The **Borrowing Revenue — Earned vs Collected** chart and `borrowing-earned-collected-chart.tsx` are removed from the revenue page; related tests are deleted. Indexing for collected/receivable is unchanged—only display goes away.
> 
> The **CDP Borrowing Fees** tile keeps the protocol-share **earned** headline but drops the separate SP-yield row, **Collected** / **Accruing** labels, and the collection progress bar. One context line shows **Gross** and how much goes to the stability pool; the subtitle is shortened to **Protocol share of borrowing fees**. **Swap Fees** subtitle becomes **Protocol share of swap fees**.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit bab72093403a2d3341d927ba6ec0c5acbe41ccab. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: ui-dashboard/src/app/revenue/_components/__tests__/revenue-page-client.test.tsx [hunk 1]
```diff
@@ -215,22 +215,17 @@ describe("RevenuePageClient degraded fee states", () => {
...
-    expect(html).toContain("Accruing");
-    expect(html).toContain("$30.63");
-    expect(html).toContain('role="progressbar"');
-    expect(html).toContain('aria-valuenow="49"');
-    expect(html).toContain(
-      "Protocol share of upfront fees + interest across 2 Celo CDP markets",
-    );
+    expect(html).toContain("of which $181.87 goes to stability");
+    // Collected/receivable stay indexed but are no longer rendered.
+    expect(html).not.toContain("Accruing");
```
[the remark this answers is not in the document]
Comment (chapati23) [reply ui-dashboard/src/app/revenue/_components/__tests__/revenue-page-client.test.tsx:225]: Fixed in 08788ce1 — added explicit `not.toContain("$30.00")` / `not.toContain("$30.63")` assertions so the regression test self-documents that the fixture's collected/receivable values never render.

**Comment by chapati23** [comment]
Addressed the 1 note in 08788ce1: explicit not-rendered assertions for the fixture's collected/receivable values.
